doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,144 @@
-| NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-| c1   | RUNNING | 10.254.118.2 (eth0) | fd42:887:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
-+------+---------+---------------------+----------------------------------------------+-----------+-----------+
-```
+   b. Mark the OVN services as enabled to ensure that they are started when the machine boots:
+
+        systemctl enable ovn-central
+        systemctl enable ovn-host
+
+   c. Stop OVN for now:
+
+        systemctl stop ovn-central
+
+   d. Note down the IP address of the machine:
+
+        ip -4 a
+
+1. On the fourth machine, install only `ovn-host` and make sure it is enabled:
+
+        sudo apt install ovn-host nano
+        systemctl enable ovn-host
+
+1. On the first three machines, configure OVN:
+
+   a. Open `/etc/defaul/ovn-central` for editing.
+
+   b. Paste in the following configuration (replace `<server_1>`, `<server_2>` and `<server_3>` with the IP addresses of the respective machines, and `<local>` with the IP address of the machine that you are on):
+
+        OVN_CTL_OPTS= \
+          --db-nb-addr=<server_1> \
+          --db-nb-create-insecure-remote=yes \
+          --db-sb-addr=<server_1> \
+          --db-sb-create-insecure-remote=yes \
+          --db-nb-cluster-local-addr=<local> \
+          --db-sb-cluster-local-addr=<local> \
+          --ovn-northd-nb-db=tcp:<server_1>:6641,tcp:<server_2>:6641,tcp:<server_3>:6641 \
+          --ovn-northd-sb-db=tcp:<server_1>:6642,tcp:<server_2>:6642,tcp:<server_3>:6642
+
+   c. Start OVN:
+
+        systemctl start ovn-central
+
+1. On all four machines, configure Open vSwitch (replace the variables as described above):
+
+        sudo ovs-vsctl set open_vswitch . external_ids:ovn-remote=tcp:<server_1>:6642,tcp:<server_2>:6642,tcp:<server_3>:6642 external_ids:ovn-encap-type=geneve external_ids:ovn-encap-ip=<local>
+
+1. Create a LXD cluster by running `lxd init` on all machines.
+   On the first machine, create the cluster.
+   Then join the other machines with tokens by running `lxc cluster add <machine_name>` on the first machine and specifying the token when initializing LXD on the other machine.
+1. On the first machine, create and configure the uplink network:
+
+        lxc network create UPLINK --type=physical parent=<bridge> --target=<machine_name_1>
+        lxc network create UPLINK --type=physical parent=<bridge> --target=<machine_name_2>
+        lxc network create UPLINK --type=physical parent=<bridge> --target=<machine_name_3>
+        lxc network create UPLINK --type=physical parent=<bridge> --target=<machine_name_4>
+        lxc network create UPLINK --type=physical
+        lxc network set UPLINK ipv4.ovn.ranges=<IP_range> ipv6.ovn.ranges=<IP_range>
```

Comment:
Can you remove the `lxc network set` lines here and move them onto the last `create` line above (possibly using the `\` line break notation for readability) so its all done in one step.